tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -167,8 +167,8 @@
 									name: 'wp-debug-log',
 								},
 								{
-									label: 'Multisite based on subdomains',
-									name: 'subdomain_multisite',
+									label: 'Enable Sandbox Access',
```

Comment:
I wonder if we should be more explicit?
```suggestion
									label: 'Enable WordPress.com Sandbox Access',
```